### PR TITLE
Add Home/Help/About pages

### PR DESCRIPTION
**Pull request by agalkiewicz**

### CODE REVIEW

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (mmarcinkowski) [inline app/assets/javascripts/static_pages.coffee]: Remove comments

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
+// They will automatically be included in application.css.
```
Comment (mmarcinkowski) [inline app/assets/stylesheets/static_pages.scss]: Same as above

File: app/controllers/static_pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
```
Comment (mmarcinkowski) [inline app/controllers/static_pages_controller.rb]: what do u think about `def home; end`

File: config/routes.rb [hunk 4]
```diff
@@ -1,58 +1,6 @@
...
-  #     post 'toggle'
-  #   end
-  #   resources :posts, concerns: :toggleable
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
```
Comment (mmarcinkowski) [inline config/routes.rb]: :+1:

File: app/assets/javascripts/static_pages.coffee (hunk 1 above)
Comment (agalkiewicz) [inline app/assets/javascripts/static_pages.coffee]: done

File: app/assets/stylesheets/static_pages.scss (hunk 2 above)
Comment (agalkiewicz) [inline app/assets/stylesheets/static_pages.scss]: done

File: app/controllers/static_pages_controller.rb (hunk 3 above)
Comment (agalkiewicz) [inline app/controllers/static_pages_controller.rb]: changed

File: test/test_helper.rb [hunk 5]
```diff
@@ -1,9 +1,12 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
+require "minitest/reporters"
+Minitest::Reporters.use!
 
 class ActiveSupport::TestCase
-  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical
+  # order.
```
Comment (mmarcinkowski) [inline test/test_helper.rb]: btw why did u split a comment into two lines ? :wink:

Comment (agalkiewicz) [inline test/test_helper.rb]: I don't know how it happened. :)

**Comment by mmarcinkowski** [comment]
@agalkiewicz please remove the branch

**Pull request merged**